src/palace/manager/sqlalchemy/model/time_tracking.py

```diff
@@ -74,12 +74,15 @@ class PlaytimeEntry(Base):
     collection: Mapped[Collection] = relationship("Collection", uselist=False)
     library: Mapped[Library] = relationship("Library", uselist=False)
 
+    loan_identifier = Column(String(50), nullable=False)
```

Comment:
Do we need 50 characters here? I don't think the value returned by `resolve_loan_identifier` should ever be more than 40.